spring-cloud-stream-module-launcher/src/test/java/org/springframework/cloud/stream/module/launcher/DefaultArgumentsNamespacingStrategyTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.launcher;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.junit.Test;
+
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.core.env.MutablePropertySources;
+
+/**
+ * Tests for DefaultArgumentsNamespacingStrategy.
+ *
+ * @author Eric Bottard
+ */
+public class DefaultArgumentsNamespacingStrategyTest {
+
+	@Test
+	@SuppressWarnings("unchecked")
+	public void testInverse() {
+		DefaultArgumentsNamespacingStrategy strategy = new DefaultArgumentsNamespacingStrategy();
+		String module = "org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT";
+
+		Map<String, ?> qualified = strategy.qualify(module, Collections.singletonMap("foo", "bar"));
```

Comment:
Well, since anyone can invoke the module launcher independently, I think that having the namespacing format explained and illustrated clearly would help with usage.